### DISCUSSION TITLE
Stacking Images with different central coordinates

### DISCUSSION THREAD

**Discussion by suvadip31** [iraf-community/iraf#145] [Q&A]
I want to stack multiple images of a particular galaxy using IRAF but the central coordinates of the images are different. How can I stack them properly so that they have the same WCS coordinates and same central coordinates? Any help is appreciated.

**Comment by olebole** [comment]
I would really not recommend to start a new project with IRAF anymore; this is really old and outdated software, and the documentation is not the best. Instead, try to dive into the astropy universe; this is much more future-proof, and there are many more people able to help you with problems.
